### DISCUSSION TITLE
network/linux-kernel-networking

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#135] [General]
# network/linux-kernel-networking

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/network/linux-kernel-networking.html

**Comment by yshdzw** [comment]
图 3-5 应该是 OUTPUT --> routing  --> POSTROUTING

**Reply by isno** [reply]
谢谢，已修正。

**Comment by DrYaoHongBin** [comment]
图3-1好像没了

**Reply by wzh469649067** [reply]
在前面那页，这是3.2了
